test/mvnormal.jl

```diff
@@ -16,9 +16,6 @@ function test_mvnormal(g::AbstractMvNormal, n_tsamples::Int=10^6,
     d = length(g)
     μ = mean(g)
     Σ = cov(g)
-    @test partype(g) == Float64
-    @test isa(μ, Vector{Float64})
```

Comment:
Maybe keep this but adjust the type as necessary?

Reply:
the only correct thing to test for is `typeof(mean(g)) === corresponding_type_parameter`, which is a bit odd to put

Reply:
Fair point